ui/app/templates/components/gutter-menu.hbs

```diff
@@ -52,33 +52,35 @@
       </p>
       <ul class="menu-list">
         <li>
-          {{#link-to "jobs"
-            (query-params jobNamespace=system.activeNamespace.id)
-            activeClass="is-active"
-            data-test-gutter-link="jobs"}}
+          <LinkTo
+            @route="jobs"
+            @query={{hash jobNamespace=system.activeNamespace.id}}
+            @activeClass="is-active"
+            data-test-gutter-link="jobs">
             Jobs
-          {{/link-to}}
+          </LinkTo>
```

Comment:
This is an example of the pattern where the invocation becomes one line longer because `@route` is not a positional argument like it was with curly bracket invocation and it seemed to me to be more consistent to have it on a separate line as with other named arguments.

Reply:
I agree with this. Consistency over line count every time.